test/DefaultCluster.Tests/TimeoutTests.cs

```diff
@@ -9,7 +9,7 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace UnitTests
+namespace DefaultCluster.Tests
 {
     // if we parallelize tests, this should run in isolation 
```

Comment:
look at this comment. This is a test that should run in isolation and cannot be parallelized, even if it runs on the default cluster. It should not be moved.